frontend/simple/views/components/GroupSettings.vue

```diff
@@ -24,15 +42,38 @@
 }
 </style>
 <script>
+import { ModalForm } from './Modals'
 import { toPercent } from '../utils/filters'
+import L from '../utils/translations'
 
 export default {
   name: 'GroupSettings',
+  components: {
+    ModalForm
+  },
   props: {
     group: Object
   },
+  data () {
+    return {
+      isActive: false
+    }
+  },
   filters: {
     toPercent
+  },
+  computed: {
+    modalTitle () { return L('New Proposal') },
+    modalSubTitle () { return L('Voting Rule to Xpto') },
+    modalSubmitText () { return L('Start Proposal') }
+  },
```

Comment:
These aren't computed properties and so shouldn't be here. Try passing them in to the modal via an SBP event or an SBP selector call.